system/jlib/jptree.ipp

```diff
@@ -623,8 +623,11 @@ public:
     void serializeCutOff(MemoryBuffer &tgt, int cutoff=-1, int depth=0);
     void deserializeSelf(MemoryBuffer &src);
     void serializeAttributes(MemoryBuffer &tgt);
-    IPropertyTree *clone(IPropertyTree &srcTree, bool self=false, bool sub=true);
-    void clone(IPropertyTree &srcTree, IPropertyTree &dstTree, bool sub=true);
+
+    void cloneIntoSelf(const IPropertyTree &srcTree, bool sub);             // clone the name and contents of srcTree into "this" tree
+    IPropertyTree * createAndClone(const IPropertyTree &srcTree, bool sub); // create a node (that matches the type of this) and clone the source
```

Comment:
personally, I'd have kept this called 'clone'. I think ipt->clone(src) is quite intuitive (despite previous 'self' etc confusing) -
```
    IPropertyTree * clone(const IPropertyTree &srcTree, bool sub); // create a node (that matches the type of this) and clone the source
    void cloneIntoSelf(const IPropertyTree &srcTree, bool sub);    // clone the name and contents of srcTree into "this" tree
    void cloneContents(const IPropertyTree &srcTree, bool sub);
```